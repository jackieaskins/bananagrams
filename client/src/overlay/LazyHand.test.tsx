import { mount } from 'enzyme';
import { Suspense } from 'react';

import LazyHand from './LazyHand';

describe('<LazyHand />', () => {
  const renderComponent = (propOverrides = {}) =>
    mount(
      <Suspense fallback="Loading">
        <LazyHand {...propOverrides} />
      </Suspense>
    );

  it('renders properly', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});
