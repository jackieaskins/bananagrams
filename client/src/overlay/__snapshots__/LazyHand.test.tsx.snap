// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LazyHand /> renders properly 1`] = `
<Suspense
  fallback="Loading"
>
  Loading
</Suspense>
`;
